### PR TITLE
created : list component

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
List component
I have added the list component in component library. So ignore other files. Focus only on the list.html , list.css (in list folder ) , (in components) .

What not to check
Responsiveness

### CODE REVIEW

File: components/list/list.html [hunk 1]
```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Neos-UI | Toast</title>
```
Comment (Muzammil-khan710) [inline components/list/list.html]: This can be changed?

**Review by Muzammil-khan710 — APPROVED** [review]
Appreciation: UI is awesome and also code is well maintained and readable
Good to merge !

File: components/list/list.html (hunk 1 above)
Comment (Shubhamkr2610) [reply components/list/list.html]: good pick , thanks . will change it

File: components/list/list.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+@import url("/components/sidebar.css");
+
+body {
+    background-image: none;
+}
+
+.component-container {
+    margin-left: 5rem;
+
+}
```
Comment (developer-anand) [inline components/list/list.css:11]: be consistent with spacing.

**Review by developer-anand — APPROVED** [review]
Can be merged

File: components/list/list.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+}
+
+#list-square {
+    list-style-type: square;
+}
+
+#list-circle {
```
Comment (aiman-ansari) [inline components/list/list.css:36]: Good naming convention

**Review by aiman-ansari — APPROVED** [review]
- Be consistent on spacing.
- naming conventions are meaningful and easy.

**Pull request merged**